### ISSUE TITLE
Workload Query

### ISSUE THREAD

**Issue by hsalahi** [brianfrankcooper/YCSB#396]
Hi there!

I have a question about CoreWorkloads loacted in /ycsb/workloads. Is it possible to access to query(s) of each workload? Where are they defined?

Thanks

**Comment by cmatser** [comment]
You can view the workload files to see which properties are defined.

Check this link to the documented description of the core workloads: https://github.com/brianfrankcooper/YCSB/wiki/Core-Workloads

**Comment by hsalahi** [comment]
Thanks for the answer. I've already read the documentation. So let me rephrase my question. In Apache JMeter database performance test case, user can define explicit query but is it possible in YCSB?

**Comment by cmatser** [comment]
I haven't used JMeter, but if I understand your question correctly, the answer is no.  You cannot define the query that will be used.  You can use various properties to control the proportion of reads.  As well, you can change the request distribution from uniform, zipfian, latest, or hotspot.  All of these reads query on the "key" of the record.

If you want a different query, you would have to augment the code manually.  YCSB would give you a starting point for performing multi-threaded operations and reporting the performance results.
